server/src/session.ts

```diff
@@ -457,6 +461,13 @@ export class Session {
         this.connection.console.info(msg);
         project.log(msg);
       }
+
+      return;
+    }
+    // The language service should be enabled at this point.
+    if (!this.languageServiceEnabled.get(project)) {
```

Comment:
Do we actually need to do this check?
At this point we already know that the language service is enabled, so we could just go ahead and print the statement.
Is this check to make sure that we don't print an activation message for a particular project twice?

Reply:
Yeah, that was the idea. But I guess it's fine to print it multiple times. Will update